VideoApp/VideoApp/Stores/RemoteConfig/RemoteConfigStore.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Copyright (C) 2020 Twilio, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+protocol RemoteConfigStoreReading: AnyObject {
+    var roomType: CommunityCreateTwilioAccessTokenResponse.RoomType { get }
+}
+
+protocol RemoteConfigStoreWriting: RemoteConfigStoreReading {
+    var roomType: CommunityCreateTwilioAccessTokenResponse.RoomType { get set }
+}
+
+class RemoteConfigStore: RemoteConfigStoreWriting {
```

Comment:
I needed to create this so I could reuse the settings update logic since now we want to save room type after sign in. Before the backend only returned room type when `createRoom = true` (fetching the Twilio access token to join a room). 

We are making backend changes to always return room type so that we know the room type (and derived video capture configuration) before navigating to the lobby screen and showing video preview.